src/dmd/dmdparams.d

```diff
@@ -0,0 +1,30 @@
+/**
+ * DMD-specific parameters.
+ *
+ * Copyright:   Copyright (C) 1999-2022 by The D Language Foundation, All Rights Reserved
+ * Authors:     $(LINK2 https://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 https://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/dmdparams.d, _dmdparams.d)
+ * Documentation:  https://dlang.org/phobos/dmd_dmdparams.html
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/dmdparams.d
+ */
+
+module dmd.dmdparams;
```

Comment:
Yes, there's redundancy in the name, it is deliberate to make it clear that this has nothing to do with the DMD front-end, and so should never be imported by front-end code.